CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to the "vscode-knative" extension will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.2.0 (2022/07/21)
+
```

Comment:
@sudhirverma Also add bump version of func cli to 0.25.0

Reply:
done